backend/tracim_backend/lib/search/elasticsearch_search/elasticsearch_search.py

```diff
@@ -487,13 +606,33 @@ def _create_ingest_pipeline(self) -> None:
             body={
                 "description": "Extract attachment information",
                 "processors": [
+                    {"remove": {"field": FILE_PIPELINE_SOURCE_FIELD}},
                     {
                         "attachment": {
                             "field": FILE_PIPELINE_SOURCE_FIELD,
                             "target_field": FILE_PIPELINE_DESTINATION_FIELD,
                         }
                     },
-                    {"remove": {"field": FILE_PIPELINE_SOURCE_FIELD}},
+                    {
+                        "set": {
+                            "if": "ctx.file_data.language == 'fr'",
+                            "field": "file_data.content_fr",
+                            "value": "{{file_data.content}}",
+                        }
+                    },
+                    {
+                        "set": {
+                            "if": "ctx.file_data.language == 'en'",
+                            "field": "file_data.content_en",
+                            "value": "{{file_data.content}}",
+                        }
+                    },
+                    {
+                        "remove": {
+                            "if": "ctx.file_data.language == 'en' || ctx.file_data.language == 'fr'",
```

Comment:
Note: i suggest either to:
- refactor this code to make add support for new language simpler
- make a todo about this.

If you do such refactor, it may worth it to add:
```
content_pt = Text(analyzer="portuguese")
content_de = Text(analyzer="german")
```
as theses lang are/will be supported in tracim.
https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-lang-analyzer.html